### ISSUE TITLE
Does not work properly on Ubuntu on Windows

### ISSUE THREAD

**Issue by raeesiqbal** [facebook/hhvm#6968]
**[Ubuntu on Windows](https://msdn.microsoft.com/commandline/wsl/about)** is Ubuntu UserMode running on Windows 10 (a new feature for developers by Microsoft) via translating Linux syscalls to Windows native syscalls. It is supposed to work for most CLI applications, as you can run Linux native ELF binaries with it. Linux native Redis works out of the box on it.
HHVM server mode and interactive mode dont seem to work at the time. But I think there should be very small modification required to make it work.

### HHVM Version
```
HipHop VM 3.13.0 (rel)
Compiler: tags/HHVM-3.13.0-0-g5b0e52b83f1dc0aa0c1dfe3a5687995c9693f5a6
Repo schema: 5fa42b79ea61b0af8d3fe1af48e0308a834bd224
```
### How to reproduce the problem
* [Install the Bash on Ubuntu on Windows](https://msdn.microsoft.com/en-us/commandline/wsl/install_guide)
* [Install HHVM](https://docs.hhvm.com/hhvm/installation/linux#ubuntu-14.04-trusty) on it

1. Run `hhvm -a`
2. Run `hhvm -m daemon -d hhvm.server.type=proxygen -d hhvm.server.port=8080 hhvm.log.file="/root/hhvm.log"`

### Expected result
1. It should start HHVM interactive mode
2. It should start HHVM server mode (:D)

### Actual result
1. It outputs this:
```
Welcome to HipHop Debugger!
Type "help" or "?" for a complete list of commands.

Note: no server specified, debugging local scripts only.
If you want to connect to a server, launch with "-h" or use:
  [m]achine [c]onnect <servername>

Unable to communicate with server. Server's down?
Unable to reconnect to server, exiting.
```
2. It outputs `Log file not specified under daemon mode.\n\n`

`hh_client` nonetheless works fine.

**Comment by raeesiqbal** [comment]
Maybe we no longer need a Windows native build :D
Ref: #5460

**Comment by Orvid** [comment]
Same as #6950 for now, as 3.13.0 has known issues.

And we do still need a native Windows build, because I develop from Windows.

**Comment by Orvid** [comment]
3.13.1 should be available for 14.04, could you see what that does? If all else, running our test suite (and Folly's as well) should be a decent workout for the Linux subsystem.

**Comment by raeesiqbal** [comment]
Not fixed by upgrading to 3.13.1. Will run the tests now.

**Comment by raeesiqbal** [comment]
Here is the output for `./configure`
```
-- The C compiler identification is GNU 4.8.4
-- The CXX compiler identification is GNU 4.8.4
-- The ASM compiler identification is GNU
-- Found assembler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++
-- Check for working CXX compiler: /usr/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Performing Test IS_X64
-- Performing Test IS_X64 - Success
-- Performing Test IS_AARCH64
-- Performing Test IS_AARCH64 - Failed
-- Performing Test IS_PPC64
-- Performing Test IS_PPC64 - Failed
-- Performing Test PTHREAD_TIMEDLOCK
-- Performing Test PTHREAD_TIMEDLOCK - Failed
-- Build type not specified: cmake build type Release.
-- Found LibDL: /usr/lib/x86_64-linux-gnu/libdl.so
-- Performing Test LIBDL_NEEDS_UNDERSCORE
-- Performing Test LIBDL_NEEDS_UNDERSCORE - Failed
-- Boost version: 1.54.0
-- Found the following Boost libraries:
--   system
--   program_options
--   filesystem
--   context
-- Found LIBGLOG: /usr/lib/x86_64-linux-gnu/libglog.so
-- Could NOT find LIBINOTIFY (missing:  LIBINOTIFY_LIBRARY)
-- Found PCRE: /usr/lib/x86_64-linux-gnu/libpcre.so
-- Performing Test SYSTEM_PCRE_HAS_JIT
-- Performing Test SYSTEM_PCRE_HAS_JIT - Failed
-- System PCRE does not have JIT enabled - will use hhvm-third-party/pcre
-- Found libevent: /usr/lib/x86_64-linux-gnu/libevent.so
-- Looking for evhttp_bind_socket_with_fd
-- Looking for evhttp_bind_socket_with_fd - not found
-- Found CURL: /usr/lib/x86_64-linux-gnu/libcurl.so (found version "7.35.0")
-- Looking for curl_multi_select
-- Looking for curl_multi_select - not found
-- Looking for curl_multi_wait
-- Looking for curl_multi_wait - found
-- Found LibXml2: /usr/lib/x86_64-linux-gnu/libxml2.so (found version "2.9.1")
-- Found PkgConfig: /usr/bin/pkg-config (found version "0.26")
-- Found LIBSQLITE3: /usr/lib/x86_64-linux-gnu/libsqlite3.so
-- Found libsqlite3: /usr/lib/x86_64-linux-gnu/libsqlite3.so
-- Could NOT find DOUBLE_CONVERSION (missing:  DOUBLE_CONVERSION_LIBRARY DOUBLE_CONVERSION_INCLUDE_DIR)
-- Using third-party bundled double-conversion
-- Could NOT find LZ4 (missing:  LZ4_LIBRARY LZ4_INCLUDE_DIR)
-- Using third-party bundled LZ4
-- Could NOT find FastLZ (missing:  FASTLZ_LIBRARY FASTLZ_INCLUDE_DIR)
-- Using third-party bundled fastlz
-- Could NOT find LIBZIP (missing:  LIBZIP_LIBRARY LIBZIP_INCLUDE_DIR_ZIP LIBZIP_INCLUDE_DIR_ZIPCONF)
-- Using third-party bundled libzip
-- Found ICU header files in /usr/include/x86_64-linux-gnu
-- Found ICU libraries: /usr/lib/x86_64-linux-gnu/libicuuc.so
-- Performing Test JEMALLOC_VERSION_MINIMUM
-- Performing Test JEMALLOC_VERSION_MINIMUM - Success
-- Found jemalloc: /usr/lib/x86_64-linux-gnu/libjemalloc.so
-- Found Intel TBB
-- Found OpenSSL: /usr/lib/x86_64-linux-gnu/libssl.so;/usr/lib/x86_64-linux-gnu/libcrypto.so (found version "1.0.1f")
-- Performing Test OPENSSL_HAVE_RAND_EGD
-- Performing Test OPENSSL_HAVE_RAND_EGD - Success
-- Found ZLIB: /usr/lib/x86_64-linux-gnu/libz.so (found version "1.2.8")
-- Found ONIGURUMA: /usr/lib/libonig.so
-- Found LIBPTHREAD: /usr/lib/x86_64-linux-gnu/libpthread.so
-- Found Readline: /usr/include
-- Found Editline: /usr/lib/x86_64-linux-gnu/libedit.so
-- Found LibElf: /usr/lib/x86_64-linux-gnu/libelf.so
-- Performing Test ELF_GETSHDRSTRNDX
-- Performing Test ELF_GETSHDRSTRNDX - Success
-- Found LibDwarf: TRUE
-- Looking for dwarf_producer_init
-- Looking for dwarf_producer_init - found
-- Performing Test DW_CONST
-- Performing Test DW_CONST - Failed
-- Looking for dwarf_producer_init_c
-- Looking for dwarf_producer_init_c - found
-- Performing Test DW_CONST
-- Performing Test DW_CONST - Failed
-- libdwarf uses char* type
-- libdwarf has dwarf_producer_init_c
-- Performing Test FOLLY_HAVE_WEAK_SYMBOLS
-- Performing Test FOLLY_HAVE_WEAK_SYMBOLS - Success
-- Looking for memrchr
-- Looking for memrchr - found
Generating Release build
-- Performing Test FOLLY_IFUNC
-- Performing Test FOLLY_IFUNC - Failed
-- Looking for clock_gettime
-- Looking for clock_gettime - found
-- Looking for pthread_atfork
-- Looking for pthread_atfork - found
-- Looking for pthread_spin_lock
-- Looking for pthread_spin_lock - found
-- Performing Test FOLLY_HAVE_VLA
-- Performing Test FOLLY_HAVE_VLA - Success
-- Did not find system RE2
-- Building RE2 from third-party
-- Looking for include file stdio.h
-- Looking for include file stdio.h - found
-- Looking for include file stdlib.h
-- Looking for include file stdlib.h - found
-- Looking for include file stddef.h
-- Looking for include file stddef.h - found
-- Looking for include file assert.h
-- Looking for include file assert.h - found
-- Looking for include file memory.h
-- Looking for include file memory.h - found
-- Looking for include file strings.h
-- Looking for include file strings.h - found
-- Looking for include file string.h
-- Looking for include file string.h - found
-- Looking for strcasecmp
-- Looking for strcasecmp - found
-- Looking for strchr
-- Looking for strchr - found
-- Check if the system is big endian
-- Searching 16 bit integer
-- Looking for sys/types.h
-- Looking for sys/types.h - found
-- Looking for stdint.h
-- Looking for stdint.h - found
-- Check size of unsigned short
-- Check size of unsigned short - done
-- Using unsigned short
-- Check if the system is big endian - little endian
-- Looking for include file inttypes.h
-- Looking for include file inttypes.h - found
-- Looking for include file locale.h
-- Looking for include file locale.h - found
-- Looking for include file limits.h
-- Looking for include file limits.h - found
-- Check size of int32_t
-- Check size of int32_t - done
-- Check size of uint32_t
-- Check size of uint32_t - done
-- Looking for strtoll
-- Looking for strtoll - found
-- Looking for atoll
-- Looking for atoll - found
-- Looking for strftime
-- Looking for strftime - found
-- Looking for setlocale
-- Looking for setlocale - found
-- Performing Test FOLLY_MALLOC
-- Performing Test FOLLY_MALLOC - Success
-- Performing Test FOLLY_FUNCTEXCEPT
-- Performing Test FOLLY_FUNCTEXCEPT - Success
-- Boost version: 1.54.0
-- Found the following Boost libraries:
--   system
--   program_options
--   filesystem
--   regex
--   context
-- Boost version: 1.54.0
-- Found the following Boost libraries:
--   system
--   thread
-- Boost version: 1.54.0
-- Found the following Boost libraries:
--   thread
--   system
-- Boost version: 1.54.0
-- Found the following Boost libraries:
--   thread
--   system
-- Looking for arpa/inet.h
-- Looking for arpa/inet.h - found
-- Looking for netinet/in.h
-- Looking for netinet/in.h - found
-- Looking for stddef.h
-- Looking for stddef.h - found
-- Looking for stdint.h
-- Looking for stdint.h - found
-- Looking for string.h
-- Looking for string.h - found
-- Looking for sys/socket.h
-- Looking for sys/socket.h - found
-- Looking for sys/time.h
-- Looking for sys/time.h - found
-- Looking for unistd.h
-- Looking for unistd.h - found
-- Check size of size_t
-- Check size of size_t - done
-- Check size of ssize_t
-- Check size of ssize_t - done
-- Check size of uint8_t
-- Check size of uint8_t - done
-- Looking for AF_LOCAL
-- Looking for AF_LOCAL - found
-- Looking for PF_LOCAL
-- Looking for PF_LOCAL - found
-- Looking for memset
-- Looking for memset - found
-- Looking for socket
-- Looking for socket - found
-- Looking for strerror
-- Looking for strerror - found
-- Found libevent: /usr/lib/x86_64-linux-gnu/libevent.so
-- Looking for event_get_version_number
-- Looking for event_get_version_number - found
-- Performing Test HAVE_LIBEVENT_209
-- Performing Test HAVE_LIBEVENT_209 - Success
-- Found libevent 2.0.9+
-- Boost version: 1.54.0
-- Found the following Boost libraries:
--   context
--   regex
--   system
--   filesystem
-- Found BZip2: /usr/lib/x86_64-linux-gnu/libbz2.so (found version "1.0.6")
-- Looking for BZ2_bzCompressInit in /usr/lib/x86_64-linux-gnu/libbz2.so
-- Looking for BZ2_bzCompressInit in /usr/lib/x86_64-linux-gnu/libbz2.so - found
-- Found Readline: /usr/include
-- Found Editline: /usr/include/editline
-- Looking for dirent.h
-- Looking for dirent.h - found
-- Looking for sys/stat.h
-- Looking for sys/stat.h - found
-- Looking for unistd.h
-- Looking for unistd.h - found
-- Looking for windows.h
-- Looking for windows.h - not found
-- Looking for C++ include type_traits.h
-- Looking for C++ include type_traits.h - not found
-- Looking for C++ include bits/type_traits.h
-- Looking for C++ include bits/type_traits.h - not found
-- Looking for bcopy
-- Looking for bcopy - found
-- Looking for memmove
-- Looking for memmove - found
-- Looking for strtoq
-- Looking for strtoq - found
-- Looking for _strtoi64
-- Looking for _strtoi64 - not found
-- Check size of long long
-- Check size of long long - done
-- Check size of unsigned long long
-- Check size of unsigned long long - done
--
--
-- PCRE configuration summary:
--
--   C compiler ...................... : /usr/bin/cc
--   C++ compiler .................... : /usr/bin/c++
--   C compiler flags ................ :  -w -ftrack-macro-expansion=0 -fno-builtin-memcmp -O3 -DNDEBUG
--   C++ compiler flags .............. :  -Wno-deprecated -Wno-strict-aliasing -Wno-write-strings -Wno-invalid-offsetof -Wno-error=array-bounds -Wno-error=switch -Wno-unused-result -Wno-sign-compare -Wno-attributes -Wno-maybe-uninitialized -Wno-unused-local-typedefs -Wno-deprecated-declarations -Wno-unused-function -ftrack-macro-expansion=0 -fno-builtin-memcmp -std=gnu++11 -fno-omit-frame-pointer -fno-operator-names -Wall -Woverloaded-virtual -Werror=format-security -ffunction-sections -fdata-sections -fno-gcse -fno-canonical-system-headers -Wvla -mcrc32 -O3 -DNDEBUG --param max-inline-insns-auto=100 --param early-inlining-insns=200 --param max-early-inliner-iterations=50 --param=inline-unit-growth=200 --param=large-unit-insns=10000 -momit-leaf-frame-pointer
--
--   Build 8 bit PCRE library ........ : ON
--   Build 16 bit PCRE library ....... : OFF
--   Build 32 bit PCRE library ....... : OFF
--   Build C++ library ............... : OFF
--   Enable JIT compiling support .... : ON
--   Enable UTF support .............. : ON
--   Unicode properties .............. : ON
--   Newline char/sequence ........... : LF
--   \R matches only ANYCRLF ......... : OFF
--   EBCDIC coding ................... : OFF
--   EBCDIC coding with NL=0x25 ...... : OFF
--   Rebuild char tables ............. : OFF
--   No stack recursion .............. : OFF
--   POSIX mem threshold ............. : 10
--   Internal link size .............. : 2
--   Parentheses nest limit .......... : 250
--   Match limit ..................... : 10000000
--   Match limit recursion ........... : MATCH_LIMIT
--   Build shared libs ............... : OFF
--   Build static libs ............... : ON
--   Build pcregrep .................. : OFF
--   Enable JIT in pcregrep .......... : ON
--   Buffer size for pcregrep ........ : 20480
--   Build tests (implies pcretest  .. : OFF
--                and pcregrep)
--   Link pcregrep with libz ......... : ON
--   Link pcregrep with libbz2 ....... : ON
--   Link pcretest with libeditline .. : OFF
--   Link pcretest with libreadline .. : ON
--   Support Valgrind .................: OFF
--   Support coverage .................:
--
-- Looking for _close
-- Looking for _close - not found
-- Looking for _dup
-- Looking for _dup - not found
-- Looking for _fdopen
-- Looking for _fdopen - not found
-- Looking for _fileno
-- Looking for _fileno - not found
-- Looking for _open
-- Looking for _open - not found
-- Looking for _snprintf
-- Looking for _snprintf - not found
-- Looking for _strdup
-- Looking for _strdup - not found
-- Looking for _stricmp
-- Looking for _stricmp - not found
-- Looking for fseeko
-- Looking for fseeko - found
-- Looking for ftello
-- Looking for ftello - found
-- Looking for open
-- Looking for open - found
-- Looking for mkstemp
-- Looking for mkstemp - found
-- Looking for snprintf
-- Looking for snprintf - found
-- Looking for strdup
-- Looking for strdup - found
-- Looking for MoveFileExA
-- Looking for MoveFileExA - not found
-- Looking for include file sys/types.h
-- Looking for include file sys/types.h - found
-- Looking for include file inttypes.h
-- Looking for include file inttypes.h - found
-- Looking for include file stdint.h
-- Looking for include file stdint.h - found
-- Check size of __int8
-- Check size of __int8 - failed
-- Check size of int8_t
-- Check size of int8_t - done
-- Check size of uint8_t
-- Check size of uint8_t - done
-- Check size of __int16
-- Check size of __int16 - failed
-- Check size of int16_t
-- Check size of int16_t - done
-- Check size of uint16_t
-- Check size of uint16_t - done
-- Check size of __int32
-- Check size of __int32 - failed
-- Check size of int32_t
-- Check size of int32_t - done
-- Check size of uint32_t
-- Check size of uint32_t - done
-- Check size of __int64
-- Check size of __int64 - failed
-- Check size of int64_t
-- Check size of int64_t - done
-- Check size of uint64_t
-- Check size of uint64_t - done
-- Check size of short
-- Check size of short - done
-- Check size of int
-- Check size of int - done
-- Check size of long
-- Check size of long - done
-- Check size of long long
-- Check size of long long - done
-- Check size of off_t
-- Check size of off_t - done
-- Check size of size_t
-- Check size of size_t - done
-- Check size of ssize_t
-- Check size of ssize_t - done
-- Build type not specified: cmake build type Release.
-- Boost version: 1.54.0
-- Found the following Boost libraries:
--   system
--   program_options
--   filesystem
--   context
-- System PCRE does not have JIT enabled - will use hhvm-third-party/pcre
-- Found libevent: /usr/lib/x86_64-linux-gnu/libevent.so
-- Found libsqlite3: /usr/lib/x86_64-linux-gnu/libsqlite3.so
-- Could NOT find DOUBLE_CONVERSION (missing:  DOUBLE_CONVERSION_LIBRARY DOUBLE_CONVERSION_INCLUDE_DIR)
-- Using third-party bundled double-conversion
-- Could NOT find LZ4 (missing:  LZ4_LIBRARY LZ4_INCLUDE_DIR)
-- Using third-party bundled LZ4
-- Could NOT find FastLZ (missing:  FASTLZ_LIBRARY FASTLZ_INCLUDE_DIR)
-- Using third-party bundled fastlz
-- Could NOT find LIBZIP (missing:  LIBZIP_LIBRARY LIBZIP_INCLUDE_DIR_ZIP LIBZIP_INCLUDE_DIR_ZIPCONF)
-- Using third-party bundled libzip
-- Found ICU header files in /usr/include/x86_64-linux-gnu
-- Found ICU libraries: /usr/lib/x86_64-linux-gnu/libicuuc.so
-- Found jemalloc: /usr/lib/x86_64-linux-gnu/libjemalloc.so
-- Found Intel TBB
-- Looking for dwarf_producer_init
-- Looking for dwarf_producer_init - found
-- Performing Test DW_CONST
-- Performing Test DW_CONST - Failed
-- Looking for dwarf_producer_init_c
-- Looking for dwarf_producer_init_c - found
-- Performing Test DW_CONST
-- Performing Test DW_CONST - Failed
-- libdwarf uses char* type
-- libdwarf has dwarf_producer_init_c
Generating Release build
-- Found ocamlc: /usr/bin/ocamlc
-- Found ocamlc.opt: /usr/bin/ocamlc.opt
-- Could NOT find LZ4 (missing:  LZ4_LIBRARY LZ4_INCLUDE_DIR)
-- Using third-party bundled LZ4
-- Looking for memfd_create
-- Looking for memfd_create - not found
-- Performing Test HAVE_DECL___NR_MEMFD_CREATE
-- Performing Test HAVE_DECL___NR_MEMFD_CREATE - Failed
-- Using built-in JSON parser
-- Building the apache extension.
-- Building the apc extension.
-- Building the array extension.
-- Building the asio extension.
-- Building the Async MySQL extension.
-- Building the bcmath extension.
-- Found EXPAT: /usr/lib/x86_64-linux-gnu/libexpat.so (found version "2.1.0")
-- Building the bz2 extension.
-- Building the collections extension.
-- Building the ctype extension.
-- Building the curl extension.
-- Building the datetime extension.
-- Building the debugger extension.
-- Building the DOMDocument extension.
-- Building the enum extension.
-- Found ICU header files in /usr/include/x86_64-linux-gnu
-- Found ICU libraries: /usr/lib/x86_64-linux-gnu/libicuuc.so
-- Building the fb extension.
-- Found ICU header files in /usr/include/x86_64-linux-gnu
-- Found ICU libraries: /usr/lib/x86_64-linux-gnu/libicuuc.so
-- Building the fileinfo extension.
-- Building the filter extension.
-- GNU FriBidi not found
-- The fribidi extension was disabled because resolving the dependency 'libFribidi 0.19.6' failed.
Not building the fribidi extension.
-- Found Freetype: /usr/lib/x86_64-linux-gnu/libfreetype.so
-- Found LibJpeg: /usr/lib/x86_64-linux-gnu/libjpeg.so
-- Could NOT find LIBICONV (missing:  LIBICONV_LIBRARY)
-- Performing Test LIBICONV_CONST
-- Performing Test LIBICONV_CONST - Failed
-- Found LibPng: /usr/lib/x86_64-linux-gnu/libpng.so
-- Found LibVPX: /usr/lib/x86_64-linux-gnu/libvpx.so
-- Building the gd extension.
-- Building the generator extension.
-- Found LIBINTL: /usr/include
-- Building the gettext extension.
-- Found GMP: /usr/lib/x86_64-linux-gnu/libgmp.so
-- Building the gmp extension.
-- Building the hash extension.
-- Building the hh extension.
-- Building the hh_client extension.
-- Building the Hot Profiler extension.
-- Building the iconv extension.
-- Found ICU header files in /usr/include/x86_64-linux-gnu
-- Found ICU libraries: /usr/lib/x86_64-linux-gnu/libicuuc.so
-- Building the icu extension.
-- Found ICU header files in /usr/include/x86_64-linux-gnu
-- Found ICU libraries: /usr/lib/x86_64-linux-gnu/libicuuc.so
-- Found LibMagicWand: /usr/lib/x86_64-linux-gnu/libMagickWand.so
-- Building the imagick extension.
-- Building the imap extension.
-- Building the intervaltimer extension.
-- Building the ipc extension.
-- Building the json extension.
-- Found ldap: /usr/lib/x86_64-linux-gnu/libldap.so;/usr/lib/x86_64-linux-gnu/liblber.so
-- Building the ldap extension.
-- Building the libxml extension.
-- Building the mail extension.
-- Building the mailparse extension.
-- Building the mbstring extension.
-- Building the mcrouter extension.
-- Found mcrypt: /usr/lib/libmcrypt.so
-- Building the mcrypt extension.
-- Found mcrypt: /usr/lib/libmcrypt.so
-- Found LIBMEMCACHED: /usr/lib/x86_64-linux-gnu/libmemcached.so
-- Building the memcache extension.
-- Building the memcached extension.
-- Building the MySQL extension.
-- Building the MySQLi extension.
-- Building the objprof extension.
-- Could NOT find LibUODBC (missing:  LIBUODBC_LIBRARIES LIBUODBC_INCLUDE_DIRS)
-- The odbc extension was disabled because resolving the dependency 'libUODBC' failed.
Not building the odbc extension.
-- Building the openssl extension.
-- Building the password extension.
-- Building the pcre extension.
-- Building the pdo extension.
-- Building the pdo_mysql extension.
-- Building the pdo_sqlite extension.
-- Building the phar extension.
-- Building the posix extension.
-- Building the process extension.
-- Building the random extension.
-- Building the readline extension.
-- Building the reflection extension.
-- Building the server extension.
-- Building the session extension.
-- Building the simplexml extension.
-- Building the soap extension.
-- Building the sockets extension.
-- Building the spl extension.
-- Building the sqlite3 extension.
-- Building the std extension.
-- Building the stream extension.
-- Building the string extension.
-- Building the thread extension.
-- Building the thrift extension.
-- Building the url extension.
-- Building the wddx extension.
-- Building the xdebug extension.
-- Building the xenon extension.
-- Building the xhprof extension.
-- Building the xml extension.
-- Building the xmlreader extension.
-- Building the xmlwriter extension.
-- Found LibXslt: /usr/lib/x86_64-linux-gnu/libxslt.so (found version "1.1.28")
-- Building the xsl extension.
-- Building the zip extension.
-- Building the zlib extension.
-- Found LibYaml: /usr/lib/x86_64-linux-gnu/libyaml.so
-- Could NOT find LibNuma (missing:  LIBNUMA_LIBRARIES LIBNUMA_INCLUDE_DIRS)
-- Found gold: /usr/bin/gold
-- Performing Test NOT_REQUIRE_ATOMIC_LINKER_FLAG
-- Performing Test NOT_REQUIRE_ATOMIC_LINKER_FLAG - Success
-- Configuring done
-- Generating done
```
